mobile/src/lib/reporter.js

```diff
@@ -7,7 +7,8 @@ export const ANALYTICS_URL = `https://${__SENTRY_TOKEN__}@sentry.cozycloud.cc/2`
 
 export function getAnalyticsConfiguration () {
   return {
-    shouldSendCallback: () => getState().mobile.settings.analytics
+    shouldSendCallback: () => getState().mobile.settings.analytics,
+    environment: __DEVMODE__ ? 'development' : 'production'
```

Comment:
Why in Sentry you are 2 keys? (development and production) https://sentry.cozycloud.cc/sentry/cozy-files-v3/settings/keys/

Is the same?